magento/magento2
# Percent of product price discount not working with exclude sku condition #36528

### Description (*)
   Percent of product price discount not working with exclude sku condition

### Related Pull Requests
        1.I checked particular item matched to cart rule condition using "$rule->getConditions()->validate($item)". 

### Fixed Issues (if relevant)

1. Fixes  #36528

### Manual testing scenarios (*)
  1.Set SKU attribute 'use for promo rule conditions' true.
  2.Add coupon code test with exclude product by SKU
 3.Add few product in cart and apploy coupon code
  4.Discount should apply for all product except exclude product using sku condition.
